api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -3531,4 +3535,80 @@ public void getConcepts_shouldPassWithAndOrNotWords() {
 		Assert.assertEquals(1, searchResults.size());
 		assertThat(searchResults.get(0).getWord(), is("SALBUTAMOL INHALER NOT"));
 	}
+
+	/**
+	 * @see ConceptService#getConceptByNameOrIdOrMap(String)
+	 */
+	@Test
+	public void getConceptByNameOrIdOrMap_shouldFindAConceptByItsConceptName() {
+		String conceptName = "COUGH SYRUP";
+		Assert.assertEquals("3", conceptService.getConceptByNameOrIdOrMap(conceptName).getConceptId().toString());
```

Comment:
i am worried of the 3 here,did you set it as a conceptId  any were up for COUGH SYRUP

Reply:
There is already a concept with the name "**COUGH SYRUP**" and its id is "**3**" in the file called [StandardDataSet.xml](https://github.com/openmrs/openmrs-core/blob/master/api/src/test/resources/org/openmrs/include/standardTestDataset.xml)
I decided to use this concept as i was testing the method in getting the concept by name.

Reply:
hello **herbert**! should i resolve this!

Reply:
that line is ok,i got to see the id in the data set